main/src/library/Int32.flix

```diff
@@ -0,0 +1,88 @@
+namespace Int32 {
+
+    /**
+     * Returns the minimum number representable by an `Int32`.
+     */
+    def minValue: Int32 = - 2 ** (size() - 1)
+
+    /**
+     * Returns the maximum number representable by an `Int32`.
+     */
+    def maxValue: Int32 = (2 ** (size() - 2) - 1) * 2 + 1 // Avoid arithmetic overflow
+
+    /**
+     * Returns the number of bits used to represent an `Int32`.
+     */
+    def size: Int32 = 32
+
+    /**
+     * Returns the smaller of `x` and `y`.
+     */
+    def min(x: Int32, y: Int32): Int32 = if (x <= y) x else y
+
+    /**
+     * Returns the larger of `x` and `y`.
+     */
+    def max(x: Int32, y: Int32): Int32 = if (x >= y) x else y
+
+    /**
+     * Returns the absolute value of `x`.
+     */
+    def abs(x: Int32): Int32 = if (x >= 0) x else -x
+
+    /**
+     * Returns the distance between `x` and `y`.
+     */
+    def dist(x: Int32, y: Int32): Int32 = abs(x - y)
```

Comment:
I've updated dist to return -1 when the distance is too big for an Int32.  Thanks